src/jdk.incubator.vector/share/classes/jdk/incubator/vector/Byte128Vector.java

```diff
@@ -341,6 +341,15 @@ public final long reduceLanesToLong(VectorOperators.Associative op,
         return VectorShuffle.fromArray(VSPECIES, sa, 0);
     }
 
+    @ForceInline
+    public VectorShuffle<Byte> toShuffle() {
+        return VectorSupport.convert(VectorSupport.VECTOR_OP_CAST,
+                                     this.getClass(), ETYPE, VLENGTH,
+                                     Byte128Shuffle.class, byte.class, VLENGTH,
+                                     this, VSPECIES,
+                                     Byte128Vector::toShuffleTemplate);
```

Comment:
ditto

Reply:
Yes, can be moved to super class. Done in the updated commit.